src/main/java/kayu/commands/CommandType.java

```diff
@@ -0,0 +1,18 @@
+package kayu.commands;
+
+/**
+ * Helps {@link kayu.commands.Command} by directing the {@link kayu.service.TaskList} to the right operations.
+ */
+public enum CommandType {
+    BYE,
+    DEADLINE,
+    DELETE,
+    DONE,
+    EMPTY,
+    EVENT,
+    FIND,
+    HELP,
+    INVALID,
+    LIST,
+    TODO;
+}
```

Comment:
This class could be shifted into the Command class to make the command type variable more clear there.